src/util/Constants.ts

```diff
@@ -208,7 +208,11 @@ export enum USER_FLAGS {
   TEAM_USER = 1 << 10,
   // UNKNOWN 1 << 11 discord/discord-api-docs#1823
   SYSTEM = 1 << 12,
-  UNREAD_URGENT_MESSAGES = 1 << 13
+  UNREAD_URGENT_MESSAGES = 1 << 13,
+  BUG_HUNTER_LEVEL_2 = 1 << 14,
+  // ?? 1 << 15
```

Comment:
```suggestion
  // UNKNOWN 1 << 15
```

Reply:
underage_deleted

Reply:
undocumented discord side, no need for us to document that as well